src/libstd/path.rs

```diff
@@ -1207,12 +1238,28 @@ impl Path {
     /// Directly wrap a string slice as a `Path` slice.
     ///
     /// This is a cost-free conversion.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use std::path::Path;
+    ///
+    /// Path::new("foo.txt");
```

Comment:
nit: In other places you bind this to a variable. I guess we should do it here as well, for consistency with other examples.